### ISSUE TITLE
How to run "function" examples?

### ISSUE THREAD

**Issue by yousunny** [victorliu/S4#55]
I can run some examples by s4 fig1.lua. But for many examples like 1d\Bi_OE_18_11969_2010, I get error get poyntingflux: numg was not set

**Comment by dvincentwest** [comment]
Yes it seems many of the examples are missing some basic settings, perhaps things that are new?  Don't know.  I added the line

	S:SetNumG(3)

To the file `S4\examples\1d\Oliva_OE_19_14735_2011\fig1c.lua` after line2, and then got a new error 

    fig1c.lua:29: attempt to index field '?' (a nil value)

I am new to RCWA, S4 and lua so any suggestions about the problems with these examples would be very appreciated.

**Comment by dvincentwest** [comment]
After looking at the Basic usage tutorial, I changed the NumG to 100 from 3, and it ran the computation from -60 to 60 in increments of 1 degree.  However, all values were 0, probably still a user error, but seems like it would be useful for the examples to work as expected out of the box
